mlir/lib/CAPI/Dialect/LLVM.cpp

```diff
@@ -7,9 +7,18 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir-c/Dialect/LLVM.h"
+#include "mlir-c/IR.h"
+#include "mlir-c/Support.h"
 #include "mlir/CAPI/Registration.h"
+#include "mlir/CAPI/Wrap.h"
+#include "mlir/Dialect/LLVMIR/LLVMAttrs.h"
 #include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Dialect/LLVMIR/LLVMOpsAttrDefs.h.inc"
```

Comment:
We shouldn't ever need to explicitly include the `.inc` files. I suspect your editor adds includes automatically, which isn't something that we necessarily want in MLIR/LLVM codebase. Transitive includes and forward declarations are totally fine.